libs/fetcher.js

```diff
@@ -459,31 +459,21 @@ Fetcher.middleware = function (options) {
                     }
                     if (err) {
                         var errResponse = getErrorResponse(err);
-                        if (req.query && req.query.returnMeta) {
-                            res.status(errResponse.statusCode).json(
-                                responseFormatter(req, res, {
-                                    output: errResponse.output,
-                                    meta: meta,
-                                })
-                            );
-                        } else {
-                            res.status(errResponse.statusCode).json(
-                                responseFormatter(req, res, errResponse.output)
-                            );
-                        }
-                        return;
-                    }
-                    if (req.query.returnMeta) {
-                        res.status(meta.statusCode || 200).json(
+                        res.status(errResponse.statusCode).json(
                             responseFormatter(req, res, {
-                                data: data,
+                                output: errResponse.output,
                                 meta: meta,
                             })
                         );
-                    } else {
-                        // TODO: Remove `returnMeta` feature flag after next release
-                        res.status(meta.statusCode || 200).json(data);
+                        return;
                     }
+
+                    res.status(meta.statusCode || 200).json(
+                        responseFormatter(req, res, {
+                            data: data,
+                            meta: meta,
+                        })
```

Comment:
thanks for cleaning this TODO item up, @pablopalacios !
@redonkulus and I were talking about this recently, about why GET success case wasn't adding the meta data :)

@redonkulus, I checked our internal code using responseFormatter. this changes seems to be ok.  You wanna double check?

Reply:
We are already on v0.7.0 publicly, internally we are on fluxible-plugin-fetch@0.3.x which uses fetchr@0.5.x so this change will not affect us.

Reply:
@lingyan oh, but from what I see in the code, only the POST failed  responses were not including the meta data. Unless you use a very old fetchr version, I think you should already have meta available. Could it be that the query parser or any other middleware is modifying the query object?

Reply:
We are in fact on an old version of fetchr, we have not upgraded to the latest (as it requires regression testing across many apps).